src/wireviz/svgembed.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import re
+import base64
+from pathlib import Path
+
+mime_subtype_replacements = {'jpg': 'jpeg', 'tif': 'tiff'}
+
+def embed_svg_images(filename_in: Path, overwrite: bool = True):
+    filename_in = Path(filename_in).resolve()
+    filename_out = Path(f'{filename_in.with_suffix("")}.b64.svg')
+    images_b64 = {}  # cache base-64 encoded images
+    re_xlink=re.compile(r"xlink:href=\"(?P<URL>.*?)\"", re.IGNORECASE)
+    with open(filename_in,'r') as file_in, open(filename_out,'w') as file_out:
+        for line in file_in:
+            for xlink in re_xlink.finditer(line):
+                imgurl = xlink.group('URL')
+                if not imgurl in images_b64:
+                    if not Path(imgurl).is_absolute():  # resolve relative image path
```

Comment:
Even though resolves relatives paths relative to the input YAML file during parsing (`wireviz.py:L49) I decided to keep this feature here, to make the `embed_svg_images()` function more independent from the WireViz implementation. It would therefore resolve any other remaining paths relative to the SVG file.